app/[locale]/projects/[projectId]/dmp/[dmpid]/s/[sid]/page.tsx

```diff
@@ -74,76 +152,124 @@ const PlanOverviewSectionPage: React.FC = () => {
-                      title: question.title
-                    })}
-                    className="react-aria-Button react-aria-Button--secondary"
-                  >
-                    {question.isAnswered ? t('sections.update') : t('sections.start')}
-                  </Link>
-                </div>
+                        {section.sectionTitle}
+                      </Link>
+                    </li>
+                  ))}
+                </ul>
+              )}
+            </nav>
+
+            <div className="container">
+              <section aria-label={"Requirements"}>
+                <h4>Requirements by {plan.funder_name}</h4>
+                <p>
+                  (DUMMY DATA) The Arctic Data Center requires when submitting to the Center,
+                  include methods to create these types of data.
+                </p>
+                <p>
+                  If using proprietary formats like Excel or MATLAB, plan to
+                  convert them to open-source formats before submission. If
+                  conversion isn&apos;t possible, explain why
+                </p>
+
+                <h4>Requirements by University of California</h4>
+                <p>
+                  (DUMMY DATA) The management of data and metadata is essential for supporting
+                  research integrity, reproducibility and collaboration. This
+                  section seeks to document the types and formats of data and
+                  metadata that will be generated in your project. Properly
+                  formatted and well-documented data enhance the visibility of
+                  your research, promote collaboration among users and ensure
+                  compliance with institutional policies and guidelines.
+                </p>
               </section>
-            ))}
+
+              {questions.length === 0 ? (
+                <section className={styles.noQuestionsMessage}>
+                  <h3>No Questions Available</h3>
+                  <p>There are currently no questions in this section.</p>
+                </section>
+              ) : (
+                questions.map((question) => (
+                  <section
+                    key={question.id}
+                    className={styles.questionCard}
+                    aria-labelledby={`question-title-${question.id}`}
+                  >
+                    <div className={styles.questionHeader}>
+                      <div className={styles.questionTitle}>
+                        <h3 id={`question-title-${question.id}`}>
+                          {stripHtml(question.title)}
+                        </h3>
+                        <p aria-live="polite">
+                          <span
+                            className={styles.progressIndicator}
+                            aria-label={`Question status: ${question.isAnswered ? 'Completed' : 'Not started'}`}
+                          >
+                            <svg
```

Comment:
When adding icons, we need to add them to the `DmpIcon` component that we already have. Adding the SVG directly in the page isn't very re-usable, and cumbersome to maintain.